app/forms/supplier_user_form.rb

```diff
@@ -33,10 +32,4 @@ def save!
       ),
     )
   end
-
-private
-
-  def email_not_taken
```

Comment:
maybe instead of removing this all together we can amend the validation to be similar to delivery partners which looks like this:
```
def email_not_taken
        return if delivery_partner_id.blank?
        return unless (user = Identity.find_user_by(email:))

        if DeliveryPartnerProfile.where(user:, delivery_partner:).exists?
          errors.add(:email, :unique, message: I18n.t("errors.email.taken"))
        end
      end
```
So raise this validation if a lead provider profile already exists for the user?

Reply:
@cwrw @leandroalemao 
Updated the validation to match DP/AB forms 👍